indico/modules/events/management/controllers/settings.py

```diff
@@ -11,13 +11,18 @@
 from werkzeug.exceptions import Forbidden
 
 from indico.core import signals
+from indico.core.db import db
+from indico.core.db.sqlalchemy.util.queries import db_dates_overlap
 from indico.modules.events.management.controllers.base import RHManageEventBase
 from indico.modules.events.management.forms import (EventClassificationForm, EventContactInfoForm, EventDataForm,
                                                     EventDatesForm, EventLocationForm, EventPersonsForm)
 from indico.modules.events.management.util import flash_if_unregistered
 from indico.modules.events.management.views import WPEventSettings, render_event_management_header_right
 from indico.modules.events.operations import update_event
 from indico.modules.events.util import track_time_changes
+from indico.modules.rb import Room
```

Comment:
It works just by coincidence. Use `from indico.modules.rb.models.rooms import Room`